src/app/core/data/data.service.ts

```diff
@@ -134,7 +134,7 @@ export abstract class DataService<T extends CacheableObject> {
       first((href: string) => hasValue(href)))
       .subscribe((href: string) => {
         const request = new FindAllRequest(this.requestService.generateRequestId(), href, options);
-        if (this.resetMsToLive) {
+        if (this.resetMsToLive !== undefined) {
```

Comment:
Because this is a use case that occurs very often (and indeed because of truthy/falsy values you're better off avoiding `if(variable)` in JS), we have a method for this: [`hasValue()`](https://github.com/DSpace/dspace-angular/blob/2cbe6a6d91d2ff2fbb2da2543174426e8a120f0a/src/app/shared/empty.util.ts#L84)

Reply:
Nice.  I'll remember to use it.